apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/WebHookNotify.java

```diff
@@ -0,0 +1,57 @@
+package com.ctrip.framework.apollo.portal.component;
+
+import javax.annotation.PostConstruct;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import com.ctrip.framework.apollo.portal.entity.bo.ReleaseHistoryBO;
+import com.ctrip.framework.apollo.portal.environment.Env;
+
+/**
+ * publish webHook
+ *
+ * @author HuangSheng
+ */
+@Component
+public class WebHookNotify {
```

Comment:
Webhook is quite generic, how about `ConfigReleaseWebhookNotifier`?

```suggestion
public class ConfigReleaseWebhookNotifier {
```

Reply:
这个确实比较通用，叫这个名字不错